org.eclipse.scanning.test/src/org/eclipse/scanning/test/malcolm/AbstractCommunicationMalcolmTest.java

```diff
@@ -128,7 +128,7 @@ public void eventPerformed(MalcolmEvent<MalcolmEventBean> e) {
 	public void testAbortIdleRunnableDevice() throws Throwable {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)